reference/docs-conceptual/learn/tutorials/01-discover-powershell.md

```diff
@@ -0,0 +1,283 @@
+
+```output
+Verb        AliasPrefix Group          Description
+----        ----------- -----          -----------
+Add         a           Common         Adds a resource to a container, or attaches an item to ano…
+```
+
+The noun on the other hand, is context specific. Know that the noun should be at the end of a commands name, like with the command `Get-Process`, for example. The noun is `Process`.
+
+Nouns and verbs can be used as parameters, when you call `Get-Command`. Thereby, they can be used to filter a response. Here's an example where `-Noun` is used:
+
+```powershell
+Get-Command -Noun Process
+```
+
+with a resulting list, only containing commands whose noun part equals `Process`:
+
+```output
+CommandType     Name                                               Version    Source
+-----------     ----                                               -------    ------
+Cmdlet          Debug-Process                                      7.0.0.0    Microsoft.PowerShell.Management
+Cmdlet          Get-Process                                        7.0.0.0    Microsoft.PowerShell.Management
+Cmdlet          Start-Process                                      7.0.0.0    Microsoft.PowerShell.Management
+Cmdlet          Stop-Process                                       7.0.0.0    Microsoft.PowerShell.Management
+Cmdlet          Wait-Process                                       7.0.0.0    Microsoft.PowerShell.Management
+```
+
+### Helper cmdlets, that helps with filtering
+
+Apart from using parameters to filter, you can also use commands to help you with this task. Here's some commands that can act as filters:
+
+- **Select-Object**. It's a very versatile command that helps you pick out specific properties from one or more objects. Additionally by using it's parameters you can limit the response you get back. Here's an example of `Select-Object` being used to ask for a limited number of records:
+
+   ```powershell
+   Get-Command | Select-Object -First 3
+   ```
+
+   The result from the above is the three first commands, counted from the top. The result looks like so:
+
+   ```outline
+   CommandType     Name                                               Version    Source
+   -----------     ----                                               -------    ------
+   Alias           Add-AdlAnalyticsDataSource                         1.0.2      Az.DataLakeAnalytics
+   Alias           Add-AdlAnalyticsFirewallRule                       1.0.2      Az.DataLakeAnalytics
+   Alias           Add-AdlStoreFirewallRule                           1.3.0      Az.DataLakeStore
+   ```
+
+   It's worth looking into to this command further as it can do a lot more [docs Select-Object](https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.utility/select-object?view=powershell-7.1)
```

Comment:
```suggestion
   It's worth looking into to this command further as it can do a lot more [docs Select-Object](https://docs.microsoft.com/powershell/module/microsoft.powershell.utility/select-object?view=powershell-7.1&preserve-view=true)
```